app/controllers/ems_infra_controller.rb

```diff
@@ -33,52 +33,79 @@ def scaling
       return
     end
 
-    @count_parameters = @stack.parameters.select { |x| x.name.include?('::count') }
+    arel_table = OrchestrationStackParameter.arel_table
+    @orig_parameters = @stack.parameters.where(arel_table[:name].matches("%::count"))
```

Comment:
Using `arel_table` is ugly, but I don't have a better suggestion.

Upstream Rails really needs to come up with a more pleasant API for this. :sweat_smile: